proxy/src/main/java/com/velocitypowered/proxy/config/VelocityConfiguration.java

```diff
@@ -384,15 +398,28 @@ public String toString() {
    * @throws IOException if we could not read from the {@code path}.
    */
   public static VelocityConfiguration read(Path path) throws IOException {
+    String defaultResource = "default-velocity.toml";
     boolean mustResave = false;
     CommentedFileConfig config = CommentedFileConfig.builder(path)
-        .defaultResource("/default-velocity.toml")
+        .defaultResource(defaultResource)
         .autosave()
         .preserveInsertionOrder()
         .sync()
         .build();
     config.load();
 
+    // Create temporary default configuration
+    File tmpFile = File.createTempFile(defaultResource, null);
```

Comment:
Yikes, this is what I feared. Maybe I need to work on a proper TOML serializer for Configurate!